### PR TITLE
Add filter LabToRgb e fix return image in RgbToLab

### PR DESCRIPTION
**Pull request by alessiamarcolini**
## Description

In order to implement #322, we need to convert back from the LAB space to the RGB space.

Moreover, it fixes a bug in the filter `RgbToLab`, where by mistake the resulting Lab image was converted to a PIL image, messing up the value range and type (it should be float in the range −128 to 127).

## Types of Changes

- [ ] Core
- [x] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Fixes: 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](https://github.com/histolab/histolab/blob/master/CONTRIBUTING.md) document.
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: src/histolab/filters/image_filters.py [hunk 1]
```diff
@@ -253,7 +253,7 @@ class RgbToLab(ImageFilter):
 
     Returns
     -------
-    PIL.Image.Image
+    np.ndarray
```
Comment (ernestoarbitrio) [inline src/histolab/filters/image_filters.py:256]: this is not an Image anymore ... please change the docstring

File: src/histolab/filters/image_filters_functional.py [hunk 2]
```diff
@@ -517,13 +543,14 @@ def rgb_to_lab(
...
+        The name of the illuminant (the function is NOT case sensitive). Default is
+        "D65".
     observer : {“2”, “10”}, optional
-        The aperture angle of the observer.
+        The aperture angle of the observer. Default is "2".
 
     Returns
     -------
-    PIL.Image.Image
+    np.ndarray
```
Comment (ernestoarbitrio) [inline src/histolab/filters/image_filters_functional.py:553]: not an Image anymore

**Review by ernestoarbitrio — CHANGES_REQUESTED** [review]
2 easy comments ,... than LGTM

**Comment by alessiamarcolini** [comment]
PR #334 needs to be merged first

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -29,6 +29,7 @@ jobs:
   build:
     needs: wsi-artifacts
     runs-on: ${{ matrix.os }}
+    continue-on-error: true
```
Comment (ernestoarbitrio) [inline .github/workflows/tests.yml:32]: is this needed? I don't see why we should continue the CI if we do not have these fixtures

**Review by ernestoarbitrio — CHANGES_REQUESTED** [review]
1 single comment then LGTM

File: .github/workflows/tests.yml (hunk 3 above)
Comment (alessiamarcolini) [reply .github/workflows/tests.yml]: yes you're right, leftovers

**Review by ernestoarbitrio — APPROVED** [review]

**Pull request merged**